tests/node/test_get_peers.py

```diff
@@ -0,0 +1,170 @@
+"""
+Provide tests for command line interface's node get peers command.
+"""
+import json
+
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_node_get_peers_configs():
```

Comment:
```suggestion
def test_node_get_peers():
```
Remove all `configs` everywhere.

Reply:
@anastasiia-bilova, done.